localization/src/languages/ko.json

```diff
@@ -53,7 +53,8 @@
             "CipherKeys": "암호화 키",
             "EnterKey": "수동으로 암호화 키 입력",
             "PickCipher": "암호화 키에서 선택",
-            "Prompt": "암호화 키 선택:"
+            "Prompt": "암호화 키 선택:",
+            "UseKey": "Use key"
```

Comment:
Can you ask @kkkhesha to translate, please?

Reply:
```suggestion
            "UseKey": "키 이용"
```

Reply:
@sertony